### PR TITLE
added CONTRIBUTING guide

### PR DESCRIPTION
**Pull request by VickyRampin**
References #12 

I figure that we should have a contributing file in this repository as well as reproduciblescience.org. Adding the corresponding web page is my next step after we approve the content of this one.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Contributing
+
+We would absolutely love any pull requests to contribute materials to our RSS feed of curated materials about reproducibility. If you add resources to our feed, the contributions will be tweeted out via the [@ReproFeed](https://twitter.com/ReproFeed) twitter bot, and added to the directory on [reproduciblescience.org](https://reproduciblescience.org/directory).
+
+## Best Practices for Contributing
+* Make sure that the resource you are suggesting isn't already in the feed. 
+* Make sure the resource you are suggesting is open access. This means that anyone should be able to access, read, and/or download it without having to pay money or login.
```
Comment (remram44) [inline CONTRIBUTING.md]: I like this requirement! The RSS definitely shouldn't be sending people to paywalls.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+* Be sure to add at least one tag to the resource. You can find a list of tags in the README file in this repository.
+
+## Workflow
+The following was adapted from [ProjectPorcupine's](https://github.com/TeamPorcupine/ProjectPorcupine)'s [CONTRIBUTING.md](https://github.com/TeamPorcupine/ProjectPorcupine/blob/master/CONTRIBUTING.md).
```
Comment (remram44) [inline CONTRIBUTING.md]: I think it's fine to have people edit from the web interface, i.e. just point them to https://github.com/ViDA-NYU/reproducibility-news/edit/master/news.yaml

screenshot_20180621_174226

Comment (VickyRampin) [reply CONTRIBUTING.md]: Ok, I can add that then. Hold on.

Comment (remram44) [reply CONTRIBUTING.md]: I mean instead of all the Git steps :wink:

**Comment by VickyRampin** [comment]
Those are good points -- I've updated accordingly (to only explain the workflow with the web GUI).

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+We would absolutely love any pull requests to contribute materials to our RSS feed of curated materials about reproducibility. If you add resources to our feed, the contributions will be tweeted out via the [@ReproFeed](https://twitter.com/ReproFeed) twitter bot, and added to the directory on [reproduciblescience.org](https://reproduciblescience.org/directory).
+
+## Best Practices for Contributing
+* Make sure that the resource you are suggesting isn't already in the feed. 
+* Make sure the resource you are suggesting is open access. This means that anyone should be able to access, read, and/or download it without having to pay money or login.
+* Be sure to add at least one tag to the resource. You can find a list of tags in the README file in this repository.
```
Comment (remram44) [inline CONTRIBUTING.md]: We can link to that

**Pull request merged**